### PR TITLE
Add pango

### PR DESCRIPTION
**Pull request by anutosh491**
Rebase on #1396 #1397 #1395

### CODE REVIEW

File: recipes/recipes_emscripten/pango/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - harfbuzz >=10.0.0
+    - fontconfig >=2.15.0
+    - freetype
+    - libpng
+    - graphite2
+    - libffi
+    - expat
+
+# tests:
+# - script:
```
Comment (anutosh491) [inline recipes/recipes_emscripten/pango/recipe.yaml:48]: Add tests !

File: recipes/recipes_emscripten/pango/build.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ex
+
+# get meson to find pkg-config when cross compiling
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
+export PKG_CONFIG_PATH_FOR_BUILD=$BUILD_PREFIX/lib/pkgconfig
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$BUILD_PREFIX/lib/pkgconfig
```
Comment (anutosh491) [inline recipes/recipes_emscripten/pango/build.sh:8]: Some difficulty finding pkg-config as mentioned here 
https://github.com/conda-forge/pango-feedstock/blob/be6e49ecd539630ee25697f0d35762d5cc884f6f/recipe/build.sh#L14-L22

File: recipes/recipes_emscripten/pango/build.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+meson_config_args=(
+    -Dintrospection=disabled # requires gobject-introspection as run-time dep
+    -Dfontconfig=enabled
+    -Dfreetype=enabled
+    -Dcairo=disabled
```
Comment (anutosh491) [inline recipes/recipes_emscripten/pango/build.sh:14]: Add TODO, try enabling cairo through a subsequent PR (some error comes up with this enabled, needs to be debugged)

Comment (anutosh491) [reply recipes/recipes_emscripten/pango/build.sh:14]: Done

**Review by IsabelParedes — APPROVED** [review]

**Pull request merged**